src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] As per the project requirement, you need to make the values of the action types match the duck pattern file structure path of the redux bookStore. Kindly refer to the project requirements documentation [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md). :pray: